apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/src/welcome-tour/style-tour.scss

```diff
@@ -221,3 +215,13 @@ $welcome-tour-card-media-extra-padding: 14%; // temporary value, to match the pa
 	max-width: 100%;
 	width: 100%;
 }
+
+.tour-kit-frame.is-mobile {
+	.tour-kit-step__welcome, .tour-kit-step__addblock, .tour-kit-step__moreoptions, .tour-kit-step__moveblock {
+		 .components-card__media img {
+			left: $welcome-tour-card-media-extra-padding;
+			top: $welcome-tour-card-media-extra-padding;
+			width: 100% - $welcome-tour-card-media-extra-padding;
+		}
+	}
+}
```

Comment:
This need to be refactored to be inside .welcome-tour-card.

Something like:
```
.welcome-tour-card {
	.tour-kit-frame.is-mobile .tour-kit-step__welcome &{
		.components-card__media img {
```

Reply:
Instead of `tour-kit-frame` let's use the custom class we define for the implementation (`wpcom-editor-welcome-tour`). So this would be something like:

```
.wpcom-editor-welcome-tour.is-mobile {
    .tour-kit-step__welcome etc { 
```

p.s. but thinking now whether we should use this parent custom class (e.g. `wpcom-editor-welcome-tour` here) to also create a step class for the components in [the other PR](https://github.com/Automattic/wp-calypso/pull/59450) e.g. introduce a  `wpcom-editor-welcome-tour__step` class there. So an implementation really never needs to use the core tour-kit classes in its styles if it doesn't want to (better guarded for backward compatibility).

So we could end up with the following patterns:


```
.wpcom-editor-welcome-tour.is-mobile {
    .wpcom-editor-welcome-tour__step.is-step-0 etc
```

and if we use the `slug` derived classes we talked about, it might make more sense with fewer concerns:

```
.wpcom-editor-welcome-tour.is-mobile {
    .wpcom-editor-welcome-tour__step-welcome etc
```